pipelines/build/prTester/pr_test_pipeline.groovy

```diff
@@ -125,10 +138,6 @@ Map<String, ?> defaultTestConfigurations = [
         "hotspot",
         "openj9"
     ],
-    "aarch64Linux": [
```

Comment:
is there a reason for removing aarch64?

Reply:
Oopsie. Must have come from the branch before this. Will fix